brands/amenity/cafe.json

```diff
@@ -113,6 +113,19 @@
       "takeaway": "yes"
     }
   },
+  "amenity/cafe|BreadTalk": {
+    "countryCodes": ["sg"],
+    "matchNames": ["Bread Talk"],
+    "tags": {
+      "amenity": "cafe",
+      "brand": "BreadTalk",
+      "brand:wikidata": "Q1106640",
+      "brand:wikipedia": "en:BreadTalk",
+      "cuisine": "coffee_shop",
```

Comment:
I'm not familiar with this brand.. Is it definitely an `amenity=cafe` + `cuisine=coffee_shop` ?

From looking around https://www.breadtalk.com.sg/   - It looks like maybe more of a `amenity=fast_food` or `shop=bakery` ...  Cafes are really for coffee places like Starbucks.

Reply:
I stand corrected. You are right that BreadTalk is primarily a bakery. Thank you for the correction.